modules/services/cloud-bench/trust_relationship/main.tf

```diff
@@ -75,19 +77,37 @@ resource "google_service_account_iam_binding" "sa_pool_binding" {
 # See https://cloud.google.com/iam/docs/access-resources-aws
 ###################################################
 
+
+data "google_iam_workload_identity_pool" "pool" {
+  project = var.project_id
+
+  provider                  = google-beta
+  workload_identity_pool_id = local.workload_identity_pool_name
+}
+
+data "google_iam_workload_identity_pool_provider" "pool_provider" {
+  project = var.project_id
+
+  provider                           = google-beta
+  workload_identity_pool_id          = local.workload_identity_pool_id
+  workload_identity_pool_provider_id = local.workload_identity_pool_name
+}
```

Comment:
this part seems not to be used, and tflint complains

Reply:
Done